### ISSUE TITLE
AccessViolationException at the textureUploadHeap.WriteToSubresource() call.

### ISSUE THREAD

**Issue by plynkus** [sharpdx/SharpDX-Samples#31]
(I'll look at this further at some point soon, but wanted to file it before I forgot.)

Observed:  System.AccessViolationException thrown during HelloTexture's texture upload.

No deltas from commit 00d5b77050d20ad6131f5680b89946aa7b9b7616 required to reproduce.
Built/run against SharpDX 3.0.0beta01.
Windows 10 build 10532.
Intel HD Graphics 5500.

**Comment by Gavin-Williams** [comment]
Me too. with the 3.0.0 beta1 nuget packages

**Comment by Syncaidius** [comment]
Same problem here too.
Built/run against SharpDX 3.0.0beta01.
Windows 10 build 10240.
AMD Radeon HD 7850.

I've also tried the latest sharpDX build, same problem.

**Comment by RobyDX** [comment]
Hi, I've just tested it with latest build (3.0.0 beta1 from web site), and work on my machine. I've installed Windows 10 SDK. Did you have it?

**Comment by Syncaidius** [comment]
Yeah, its installed. The rest of the samples work fine. Its only in the HelloTexture sample that it happens.

I've tried updating the SDK too, but apparently I already have the latest.
